packaging/installer/functions.sh

```diff
@@ -477,20 +480,29 @@ restart_netdata() {
 
 	local started=0
 
-	progress "Start netdata"
+	progress "Restarting netdata instance"
 
 	if [ "${UID}" -eq 0 ]; then
-		service netdata stop
-		stop_all_netdata
-		service netdata restart && started=1
+		echo >&2
+		echo >&2 "Stopping all netdata threads"
+		run stop_all_netdata
+		local start_cmd="${NETDATA_START_CMD}"
+
+		if [ "$(uname 2> /dev/null)" == "FreeBSD" ]; then
+			start_cmd="${NETDATA_START_CMD/ start/ onestart}"
```

Comment:
hm... I now see what you meant.
It is probably better to set 2 variables, at the point `NETDATA_START_CMF` is set:

```sh
NETDATA_START_CMD="the command to start netdata from the installer"
NETDATA_USER_START_CMD="the command a user should use to start netdata"
```

Reply:
ack